webpack/assets/javascripts/react_app/components/ForemanModal/subcomponents/SubmitOrCancel/SubmitOrCancelActions.js

```diff
@@ -24,16 +28,15 @@ export const submitModal = ({
   message,
   method = 'delete',
   closeFn,
-  onErr = onModalError,
-  onSucc = () => {},
-  onSubmitStop,
-  onSubmitStart,
+  onError = onModalError,
```

Comment:
The name of `onError` makes me think this is an error handler function that will automatically be called when an error is thrown, but instead it seems like it's just retrieving the error message to be displayed in the Toast.  Kind of like a selector, but instead of selecting from Redux state it's selecting from the API response.  Maybe rename to `getErrorMsg` or something similar?